src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -189,6 +189,20 @@ public void testCache() throws Exception {
             getPath("InputAbstractJavadocCache2.java"), expected);
     }
 
+    @Test
+    public void testCache2() throws Exception {
+        final String[] expected = {};
+        verifyWithInlineConfigParser(getPath("InputAbstractJavadocCache3.java"), expected);
+    }
+
+    @Test
+    public void testCache3() throws Exception {
```

Comment:
please do not follow our bad pattern of naming with digits.
It was lazy decision that bites after some time.
please make name wordy `testCacheWithTwoBlockCommentAtSameLine`
please do "same" for other test method and Inputs.

Reply:
Done